tree/ntuple/v7/inc/ROOT/RNTupleZip.hxx

```diff
@@ -56,7 +56,7 @@ public:
 
    /// Returns the size of the compressed data. Data is compressed in 16MB blocks and written
    /// piecewise using the provided writer
-   size_t operator() (const void *from, size_t nbytes, int compression, Writer_t fnWriter) {
+   size_t Zip(const void *from, size_t nbytes, int compression, Writer_t fnWriter) {
```

Comment:
I recommend explicit calling out this interface change in the commit log.

Reply:
Done